scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -134,7 +134,7 @@ package com.twitter.scalding {
     val boxedSemigroup = Externalizer(commutativeSemigroup)
 
     val DEFAULT_CACHE_SIZE = 100000
-    val SIZE_CONFIG_KEY = AggregateBy.AGGREGATE_BY_THRESHOLD
+    val SIZE_CONFIG_KEY = AggregateByProps.AGGREGATE_BY_CAPACITY
```

Comment:
This is going to break lots of users, so lets try not do this. any usage of -D on the commandline will be broken